daisy/step_create_images.go

```diff
@@ -143,17 +147,24 @@ func (c *CreateImages) run(ctx context.Context, s *Step) error {
 		wg.Add(1)
 		go func(ci *CreateImage) {
 			defer wg.Done()
-
-			project := strOr(ci.Project, w.Project)
-
 			// Get source disk link if SourceDisk is a daisy reference to a disk.
 			if d, ok := disks[w].get(ci.SourceDisk); ok {
 				ci.SourceDisk = d.link
 			}
 
+			// Delete existing if OverWrite is true.
+			if ci.OverWrite {
```

Comment:
Do we need to do this for disks and instances?

We could run into issues here when the existing disk is attached... Hmm. For now, we could let the workflow fail, I suppose.

Reply:
I specifically only implemented this for images as that could get more complicated